clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/AwsConfiguration.groovy

```diff
@@ -289,4 +300,21 @@ class AwsConfiguration {
 
     new AwsCleanupProviderSynchronizer()
   }
+
+  @Bean
+  AmazonServerGroupProvider amazonServerGroupProvider(ApplicationContext applicationContext) {
```

Comment:
This is my attempt to get around ...

```
...groovyStubs/clouddriver-aws_main/java-production/com/netflix/spinnaker/clouddriver/aws/model/AmazonCluster.java
Error:(10, 111) java: com.netflix.spinnaker.clouddriver.aws.model.AmazonCluster is not abstract and does not override abstract method getLoadBalancers() in com.netflix.spinnaker.clouddriver.model.Cluster
Error:(29, 87) java: getLoadBalancers() in com.netflix.spinnaker.clouddriver.aws.model.AmazonCluster cannot implement getLoadBalancers() in com.netflix.spinnaker.clouddriver.model.Cluster
  return type java.util.Set<com.netflix.spinnaker.clouddriver.aws.model.AmazonLoadBalancer> is not compatible with java.util.Set<com.netflix.spinnaker.clouddriver.model.LoadBalancer>
Error:(25, 86) java: getServerGroups() in com.netflix.spinnaker.clouddriver.aws.model.AmazonCluster cannot implement getServerGroups() in com.netflix.spinnaker.clouddriver.model.Cluster
  return type java.util.Set<com.netflix.spinnaker.clouddriver.aws.model.AmazonServerGroup> is not compatible with java.util.Set<com.netflix.spinnaker.clouddriver.model.ServerGroup>
```

Saw the above in both IntelliJ and Gradle whenever I attempted to inject `AmazonClusterProvider`.

Ideas?

Reply:
really no idea here, I blame groovy nonsense